utils/parallel-process-runner/src/SystemInfo.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Utils\ParallelProcessRunner;
+
+use Rector\Utils\ParallelProcessRunner\Exception\CouldNotDeterminedCpuCoresException;
+
+final class SystemInfo
+{
+    /**
+     * @see https://gist.github.com/divinity76/01ef9ca99c111565a72d3a8a6e42f7fb
+     *
+     * returns number of cpu cores
+     * Copyleft 2018, license: WTFPL
+     */
+    public function getCpuCores(): int
+    {
```

Comment:
This method is used in `ci/run_all_sets_parallel.php`
But Rector doesn't scan it in paths, so it needs to be added there:
https://github.com/rectorphp/rector/blob/master/rector-ci.yaml#L10

Reply:
full ci or only this single file?

Reply:
I'm handling it here https://github.com/rectorphp/rector/pull/3097